x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/home/index.tsx

```diff
@@ -225,6 +228,32 @@ const InstalledPackages: React.FC = memo(() => {
     return mapToCard(getAbsolutePath, getHref, item);
   });
 
+  const callout =
+    selectedCategory === 'updates_available' ? null : (
+      <Fragment>
+        <EuiSpacer />
+        <EuiCallOut
+          title={i18n.translate('xpack.fleet.epmList.availableCalloutTitle', {
+            defaultMessage: 'Only installed Elastic Agent Integrations are displayed.',
+          })}
+          iconType="iInCircle"
+        >
+          <p>
+            {i18n.translate('xpack.fleet.epmList.availableCalloutIntroText', {
+              defaultMessage:
+                'To learn more about integrations and the new Elastic Agent, read our',
+            })}{' '}
+            <EuiLink href={docLinks.links.fleet.learnMoreBlog} target="_blank">
+              {i18n.translate('xpack.fleet.epmList.availableCalloutBlogText', {
+                defaultMessage: 'announcement blog post',
+              })}
+            </EuiLink>
+            .
```

Comment:
I am not sure what is the best way to translate here, but I think it this solution could not work with some locale, it will probably be better to have something like 

```jsx
{i18n.translate('xpack.fleet.epmList.availableCalloutIntroText', {
              defaultMessage:
                'To learn more about integrations and the new Elastic Agent, read our {link}',
            }, values: { link: (<EuiLink href={docLinks.links.fleet.learnMoreBlog} target="_blank">
              {i18n.translate('xpack.fleet.epmList.availableCalloutBlogText', {
                defaultMessage: 'announcement blog post',
              })}
            </EuiLink>) })}
```

Reply:
+1 I believe this is the recommended pattern.